klippa/src/lib.rs

```diff
@@ -461,6 +464,10 @@ fn subset_table<'a>(
         Maxp::TAG => {
             subset_maxp(font, plan, builder)?;
         }
+
+        Os2::TAG => {
+            subset_os2(font, plan, builder)?;
+        }
```

Comment:
Here and above ther eseem to be a lot of extra {}, why not `Os2::TAG => subset_os2(font, plan, builder)?,`

Reply:
Most of your branches have your functions return type. It looks like you could drop the final `Ok(())` and write this function as just the match, e.g.

```rust
fn subset_table<'a>(
    tag: Tag,
    font: &FontRef<'a>,
    plan: &Plan,
    builder: &mut FontBuilder<'a>,
) -> Result<(), SubsetError> {
    match tag {
        Glyf::TAG => subset_glyf_loca(plan, font, builder),
        // head is handled by glyf table if exists
        Head::TAG => font.glyf().map(|_| ()).or_else(|_| subset_head(font, plan, builder)),            
        //Skip, handled by Hmtx
        Hhea::TAG => Ok(()),
        Hmtx::TAG => subset_hmtx_hhea(font, plan, builder),
        //Skip, handled by glyf
        Loca::TAG => Ok(()),
        Maxp::TAG => subset_maxp(font, plan, builder),
        Os2::TAG => subset_os2(font, plan, builder),
        _ => if let Some(data) = font.data_for_tag(tag) {
                builder.add_raw(tag, data);
                Ok(())
            } else {
                Err(SubsetError::SubsetTableError(tag))
            }
    }
}
```

Reply:
Thanks!